app/assets/stylesheets/semantic-ui/modules/_checkbox.scss

```diff
@@ -236,7 +232,7 @@
 .ui.checkbox input[disabled] ~ label {
```

Comment:
I think the hover effect need to be disabled when the checkbox is disabled. It is not clear that the checkbox is disabled unless you have another next to the disabled so you see the difference